packages/expo/src/provider/ClerkProvider.tsx

```diff
@@ -29,8 +29,12 @@ const SDK_METADATA = {
 };
 
 export function ClerkProvider(props: ClerkProviderProps): JSX.Element {
-  const { children, tokenCache, publishableKey, __experimental_passkeys, ...rest } = props;
+  const { children, tokenCache, publishableKey, __experimental_passkeys, experimental, ...rest } = props;
   const pk = publishableKey || process.env.EXPO_PUBLIC_CLERK_PUBLISHABLE_KEY || process.env.CLERK_PUBLISHABLE_KEY || '';
+  const rethrowOfflineNetworkErrors =
+    experimental?.rethrowOfflineNetworkErrors ||
+    process.env.EXPO_PUBLIC_CLERK_EXPERIMENTAL_RETHROW_OFFLINE_NETWORK_ERRORS ||
```

Comment:
Let's use the `isTruthy` helper from `@clerk/shared` on this env var

Reply:
Ended up removing the environment variable on Expo, as it's not needed.